src/neo-vm/OpCode.cs

```diff
@@ -441,9 +437,21 @@ public enum OpCode : byte
         /// </summary>
         FROMALTSTACK = 0x6C,
         /// <summary>
+        /// Duplicates the item on top of alt stack and put it on top of main stack.
+        /// </summary>
+        DUPFROMALTSTACK = 0x6D,
+        /// <summary>
+        /// Copies the bottom of alt stack and put it on top of main stack.
+        /// </summary> 
+        DUPFROMALTSTACKBOTTOM = 0x6E,
+        /// <summary>
+        /// Returns true if the input is null. Returns false otherwise.
+        /// </summary>
+        ISNULL = 0x70,
+        /// <summary>
         /// The item n back in the main stack is removed.
         /// </summary>
-        XDROP = 0x6D,
+        XDROP = 0x71,
```

Comment:
Is it really necessary?

Reply:
Why not?

Reply:
Since historically this has been its position...